src/components/pages/Home/Home.js

```diff
@@ -1,20 +1,41 @@
-import React from "react";
+import React, {useState, useEffect} from "react";
 import { Tabs, Tab } from "react-bootstrap";
 import AccountDashboard from "../AccountDashboard/AccountDashboard";
+import AuthService from "../../../Services/AuthService";
 import Comments from "../../Comments/Comment";
 import ProjectsList from "../../ProjectsList/ProjectsList";
 import Navbar from "../../Nav/Nav";
 import "../../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
 import "./styles/Home.css";
 
 function Home(props) {
+
+  const [user, setUser] = useState();
+  const [project, setProject] = useState(null);
+  const getData = async () => {
+    try {
+      const {data} = await AuthService.getInfo("5f5a1ef56157b937082c4229");
+      console.log(data)
+      setUser(data.firstName + " " + data.lastName);
+      setProject(data.project.length);
+    }
+    catch (err) {
+      console.log(`Error - AccountDashboard.js - ${err}`);
+    }
+  }
+  
+  useEffect(() => {
+    getData({});
+  }, []);
+  
+
   return (
-    <>
+    <React.Fragment>
       <Navbar />
       <Tabs defaultActiveKey="home" id="uncontrolled-tab-example">
         {/* eventkey is acting as the event listener the tabs won't change unless eventkey is specified but it doesn't need to be the same as the title */}
         <Tab eventKey="home" title="Dashboard">
-          <AccountDashboard />
+          <AccountDashboard  username={user} project={project} />
```

Comment:
This does not solve the problem of not being able to get user _id. We are still using a static user id to get this user information. Remove these props, and remove use of props in AccountDashboard component